salt/cloud/clouds/ec2.py

```diff
@@ -4059,6 +4061,13 @@ def create_volume(kwargs=None, call=None, wait_to_finish=False):
     # You can't set `encrypted` if you pass a snapshot
     if 'encrypted' in kwargs and 'snapshot' not in kwargs:
         params['Encrypted'] = kwargs['encrypted']
+        if ''kmskeyid' in kwargs:
```

Comment:
Derp, pushed too early. And I see one test is failing due to a documentation mismatch, do you want that in this PR as well?